docs/setup/aws.md

```diff
@@ -1,107 +1,148 @@
 # AWS setup
 
+## Warnings
+
 **NOTE that setting up Grapl *will* incur AWS charges! This can amount to hundreds of dollars a month based on the configuration.**
 This setup script is designed for testing, and may include breaking changes in future versions, increased charges in future versions, or may otherwise require manually working with CloudFormation.
 If you need a way to set up Grapl in a stable, forwards compatible manner, please get in contact with us directly.
 
-## Installing Dependencies
+## Preparation
 
-To get started, you'll need to install the following dependencies:
+### Local AWS credentials
 
-- Node
-- Typescript
-- AWS CDK: `npm i -g aws-cdk@X.Y.Z`
-  - version must be >= the version in [Grapl's package.json file](https://github.com/grapl-security/grapl/blob/main/src/js/grapl-cdk/package.json) - for instance, `@1.71.0`
-- AWS CLI: `pip install awscli`
+See full instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
 
-You'll also need to have local AWS credentials, and a configuration profile. Instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+You should have a local file `~/.aws/credentials`, with an entry resembling this format:
 
-If you intend to use Grapl's provided demo data, you'll allso need some Python3 dependencies.
-- [boto3](https://github.com/boto/boto3)
-- [zstd](https://pypi.org/project/zstd/)
+```
+[my_profile]
+aws_access_key_id=...
+aws_secret_access_key=...
+aws_session_token=...
+```
 
+You will need the **profile** to configure your account, if you haven't already:
 
-## Clone from repo
+`aws configure --profile "my_profile"`
 
-First things first, clone the repo:
-```bash
-git clone https://github.com/grapl-security/grapl.git
-```
-
-## Build deployment artifacts
+If your profile's name is not "default", then note it down, as you will need to include it as a parameter in later steps.
 
-To execute a local Grapl build, run the following in Grapl's root:
+### Installing Dependencies
 
-```bash
-TAG=$GRAPL_VERSION make zip
-```
+You'll need to have the following dependencies installed:
 
-`GRAPL_VERSION` can be any name you want. Just make note of it, we'll
-use it in the next step.
+- Node
+- TypeScript
+- AWS CDK:
+  - `npm i -g aws-cdk@X.Y.Z`
+  - version must be >= the version in [Grapl's package.json file](https://github.com/grapl-security/grapl/blob/main/src/js/grapl-cdk/package.json) - for instance, `@1.71.0`
+- AWS CLI:
+  - `pip install awscli`
```

Comment:
I'd recommend here to install the AWS CLI v2 with docker instead: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-docker.html

I added this alias to my `~/.bashrc`: 
```
alias aws='docker run --rm -it -v ~/.aws:/root/.aws -v $(pwd):/aws -e AWS_PROFILE amazon/aws-cli'
```